src/tests/ftest/cart/test_group_rpc.h

```diff
@@ -20,18 +20,22 @@
  * Any reproduction of computer software, computer software documentation, or
  * portions thereof marked with this legend must also reproduce the markings.
  */
+
+/* Set array indices into my_proto_fmt_test_group arrays */
 #define TEST_OPC_CHECKIN	CRT_PROTO_OPC(0x010000000, 0, 0)
-#define TEST_OPC_PING_DELAY	CRT_PROTO_OPC(0x010000000, 0, 2)
+#define TEST_OPC_PING_DELAY	CRT_PROTO_OPC(0x010000000, 0, 3)
+#define TEST_OPC_SWIM_STATUS	CRT_PROTO_OPC(0x010000000, 0, 2)
 #define TEST_OPC_SHUTDOWN	CRT_PROTO_OPC(0x010000000, 0, 1)
 
-
-#define CRT_ISEQ_TEST_PING_DELAY /* input fields */		 \
+/* input fields */
+#define CRT_ISEQ_TEST_PING_DELAY  				 \
```

Comment:
(style)  please, no space before tabs
(style)  Macros with complex values should be enclosed in parentheses